src/components/ui/InputText/input-text.module.scss

```
@@ -26,21 +26,21 @@
   --fs-input-transition-property   : var(--fs-transition-property);
   --fs-input-transition-timing     : var(--fs-transition-timing);
 
-  // Message
-  --fs-input-message-size          : var(--fs-text-size-legend);
-  --fs-input-message-line-height   : 1.1;
-  --fs-input-message-error-color   : var(--fs-color-danger-text);
-  --fs-input-message-margin-top    : var(--fs-spacing-0);
-
   // Label
   --fs-input-label-padding         : 0 var(--fs-spacing-2);
   --fs-input-label-color           : var(--fs-color-text-light);
   --fs-input-label-size            : var(--fs-text-size-tiny);
 
+  // Message
+  --fs-input-message-size          : var(--fs-text-size-legend);
+  --fs-input-message-line-height   : 1.1;
+  --fs-input-message-margin-top    : var(--fs-spacing-0);
+
```

WDYT about using `error-message` here too?

```suggestion
  // Error Message
  --fs-input-error-message-size        : var(--fs-text-size-legend);
  --fs-input-error-message-line-height : 1.1;
  --fs-input-error-message-margin-top  : var(--fs-spacing-0);
  --fs-input-error-message-color       : var(--fs-color-danger-text);

```